fea-rs/src/parse/grammar/metrics.rs

```diff
@@ -177,7 +177,30 @@ fn eat_location_item(parser: &mut Parser, recovery: TokenSet) -> bool {
     parser.in_node(AstKind::LocationSpecItemNode, |parser| {
         parser.eat_tag();
         parser.expect_recover(Kind::Eq, recovery.add(Kind::Comma));
-        parser.expect_recover(Kind::Number, recovery.add(Kind::Comma));
+        if !expect_axis_location(parser) {
+            parser.err_recover(
+                "expected axis location (number or float)",
+                recovery.add(Kind::Comma),
+            );
+        }
+    });
+    true
+}
+
+/// A float or integer, optionally followed by one of the suffixes 'u', 'c' or 'n'
```

Comment:
although this is not in the official spec, is there a link to some draft of the spec proposal that you could link to here?

Reply:
> the suffixes 'u', 'c' or 'n'

typo: there's no "c"; it's "u" for user, _"d"_ for design and "n" for normalized?

Reply:
oops, good catch :)

The closest thing to a proposal for this is this comment: https://github.com/harfbuzz/boring-expansion-spec/issues/94#issuecomment-1608007111